ci/checks/check_no_line_number_snippets.py

```diff
@@ -8,6 +8,15 @@
 import sys
 from typing import List
 
+ITEMS_IGNORED_FROM_LINE_NUMBER_SNIPPET_CHECKER = {
+    "docs/docusaurus/node_modules/prism-react-renderer/README.md",
```

Comment:
Just a suggestion, could we ignore the whole `/node_modules/` folder? These are packages installed by npm, not our code.

Reply:
@anthonyburdi Oops! Forgot that one here.  Will do an update PR real quick!  Thanks.